README.md

```diff
@@ -3,6 +3,12 @@
 
 The Microsoft Authentication Library for JavaScript enables client-side JavaScript web applications, running in a web browser, to authenticate users using [Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview) for work and school accounts (AAD), Microsoft personal accounts (MSA), and social identity providers like Facebook, Google, LinkedIn, Microsoft accounts, etc. through [Azure AD B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-overview#identity-providers) service. It also enables your app to get tokens to access [Microsoft Cloud](https://www.microsoft.com/enterprise) services such as [Microsoft Graph](https://graph.microsoft.io).
 
+## Library versions and support
+
+AngularJS  |  MSAL Angular   |  MSAL.js
+-----------|-----------------|-------------
+ 4.3 to 5  | 1.0.0 (Preview) |    1.3
```

Comment:
@jasonnutter  do these look right to you>

Reply:
The original question was about which version of Angular were supported by MSAL Angular. Example:

Angular versions | Library & Version
-------------------|-----------------
AngularJS < 4  | `@azure/msal-angularjs@0.1.1`
Angular 4, 5 | `@azure/msal-angular@0.1.4`
Angular > 6 | `@azure/msal-angular@1.0.0`

Reply:
And I'm not sure this should go in the root README. I think we should have a dedicated section/page to framework support.

Reply:
I think we also need to clear articulate what our support schedule looks like for the older versions of Angular. 

For example, Angular v6 just exited LTS by the Angular team. Should we still support it? https://angular.io/guide/releases#support-policy-and-schedule

Reply:
@aiwangmicrosoft I'm working on the next Angular update now, so I think it would be good for us to talk through all this as were getting ready to release that.

Reply:
@negoe Given that I'm no longer taking over JS, does it make sense to bring in Haley or you to help review with these issues?